src/lib/mina.ts

```diff
@@ -55,12 +55,39 @@ interface TransactionId {
 }
 
 interface Transaction {
+  /**
+   * Transaction structure used to describe a state transition on the Mina blockchain.
+   */
   transaction: ZkappCommand;
+  /**
+   * Returns a JSON representation of the {@link Transaction}.
+   */
   toJSON(): string;
+  /**
+   * Returns a JSON representation of only the fields that differ from the original {@link Transaction}.
+   */
   toPretty(): any;
+  /**
+   * Returns the GraphQL query for the Mina daemon.
+   */
   toGraphqlQuery(): string;
+  /**
+   * Signs all {@link AccountUpdate}s and {@link SmartContract}s included in the {@link Transaction} that require a signature.
```

Comment:
can we remove "and {@link SmartContract}s"? correct would be "SmartContract method invocations" but since they end up as AccountUpdates as well we can leave it out I think